tests/zfs-tests/tests/functional/mmp/mmp_reset_interval.ksh

```diff
@@ -58,7 +63,58 @@ log_must set_tunable64 zfs_multihost_interval $MMP_INTERVAL_DEFAULT
 uber_count=$(count_mmp_writes $TESTPOOL 1)
 
 if [ $uber_count -eq 0 ]; then
-	log_fail "mmp writes did not start when zfs_multihost_interval reduced"
+	log_fail "ERROR: mmp writes did not start when zfs_multihost_interval reduced"
 fi
 
+# 7. Verify mmp_write and mmp_fail are written
+for fails in $(seq $MMP_FAIL_INTERVALS_MIN $((MMP_FAIL_INTERVALS_MIN*2))); do
+	for interval in $(seq $MMP_INTERVAL_MIN 200 $MMP_INTERVAL_DEFAULT); do
+		log_must set_tunable64 zfs_multihost_fail_intervals $fails
+		log_must set_tunable64 zfs_multihost_interval $interval
+		log_must sync_pool $TESTPOOL
```

Comment:
Does the pool really need to be synced twice?  If so, can you add a comment explaining why.

Reply:
No, that's a mistake.  Removed the extra.